modules/apps/commerce/commerce-subscription-web/src/main/java/com/liferay/commerce/subscription/web/internal/display/context/BaseCPDefinitionSubscriptionInfoDisplayContext.java

```diff
@@ -45,6 +45,10 @@ public BaseCPDefinitionSubscriptionInfoDisplayContext(
 	}
 
 	public CPSubscriptionType getCPSubscriptionType(String subscriptionType) {
+		if (subscriptionType.isEmpty()) {
```

Comment:
Please Use Validator instead.
`if (Validator.isNull(subscriptionType)`. You can update this PR accordingly.